content/community/titanpool.md

```diff
@@ -0,0 +1,11 @@
+---
+publish: true
+image: >-
+  https://user-images.githubusercontent.com/16746274/222634706-65ae3e00-653d-4f78-8148-d59e31d48d39.png
```

Comment:
Please upload the attached logo to `cloudinary` and update the `image` url.
![Titan-Node-Logo-Medium](https://user-images.githubusercontent.com/16746274/222634706-65ae3e00-653d-4f78-8148-d59e31d48d39.png)